ciasteczka-kokosowe/README.md

```diff
@@ -0,0 +1,27 @@
+Ciasteczka ryżowo-kokosowe
+==========================
+
+_Wg przepisu Asi Czerwińskiej_
+
+Składniki
+---------
+
+* 1 opakowanie kleiku ryżowego (nie smakowego!) (?g)
+* 1 kostka masła (200 g)
+* 3 jajka (?g)
+* 1 szklanka cukru (?g)
+* wiórki kokosowe na oko - zapewne 6-8 łyżeczek (?g)
+* 1 cukier waniliowy (?g)
+* 1 łyżeczka proszku do pieczenia
+* (opcjonalnie) dżem do dekoracji
+
+Przygotowanie
+-------------
+
+* wymieszać wszystkie suche składniki
+* dodać posiekane masło,
+* dodać jajka, zagnieść ręcznie
+* uformować ciasteczka; na środku każdego można zrobić wklęśnięcie
+* piec 15-20 minut w piekarniku nagrzanym do 180 stopni (bez termoobiegu)
+* po 10 minutach zmienić temperaturę na 160 stopni i włączyć termoobieg (ten krok może nie mieć znaczenia)
```

Comment:
Przydałoby się ustalić jakie to ma znaczenie.

Reply:
W sumie to nie będę się aż tak bawił. Ważne, że w 160 stopniach z termoobiegiem wychodzą dobrze.